api/server/server.go

```diff
@@ -743,6 +744,14 @@ func WithPrometheus() Option {
 	}
 }
 
+func WithSpanConverter() Option {
```

Comment:
can this take an argument for latency buckets? Not the end of the world, but it would be good to keep buckets consistent with service side.